airbyte-config/models/src/test/java/io/airbyte/config/AirbyteProtocolConvertersTest.java

```diff
@@ -25,8 +25,10 @@
 package io.airbyte.config;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import com.google.common.collect.Lists;
+import io.airbyte.commons.enums.Enums;
```

Comment:
need to add existing unit tests here to check that the new fields are properly mapped.